docs/app/Examples/elements/Label/Variations/LabelExampleSize.js

```diff
@@ -0,0 +1,19 @@
+import _ from 'lodash'
+import React from 'react'
+import { Label } from 'stardust'
+
+const LabelExampleSize = () => {
+  const sizes = ['mini', 'tiny', 'small', 'medium', 'large', 'big', 'huge', 'massive']
```

Comment:
More info, meta used to have more keys and have more purpose. Over time, it keeps shrinking. Considering the parent, type, and name keys are only for documentation, I hope to pull them out of the code and into the doc block. We already parse the docblock for docs so grabbing type, parent, and name from there makes sense. 

This leaves only the props in the meta. In farther out, less defined goals, I'd like to better formalize how we define props to support the missing abilities of the component explorer #427.